core/storage/improvements/gae_models.py

```diff
@@ -300,6 +300,5 @@ def create(
             status=status,
             resolver_id=resolver_id,
             resolved_on=resolved_on)
-        task_entry.update_timestamps()
```

Comment:
Just to check: are all the cases where we we're removing ``update_timestamps`` & not replacing the model usage with a human maintained model because we didn't actually need to explicitly update the timestamps manually there, and can instead now rely on the automatic hook?

Reply:
Yep.